lib/bugsnex/plug.ex

```diff
@@ -75,4 +77,21 @@ defmodule Bugsnex.Plug do
     {:ok, hostname} = :inet.gethostname
     to_string(hostname)
   end
+
+  defp filter_parameters(params) do
+    do_filter(params, Application.get_env(:bugsnex, :filter_params, @default_filter_params))
+  end
+
+  defp do_filter(%{__struct__: mod} = struct, _params) when is_atom(mod), do: struct
+  defp do_filter(%{} = map, params) do
```

Comment:
same thing, first argument is actual params, second is `filter_params` or well, maybe a better name is `params_to_filter` or `param_keys_to_filter` or something like that, `params` makes me think it's the actual parameters  (which gets confusing when we check for membership)